### PR TITLE
[EAM-1953] Add other id of equipment to the titles of checklist items

### PR DESCRIPTION
**Pull request by Amfgcp**
Depends on https://github.com/cern-eam/eam-light-backend/pull/79 and https://github.com/cern-eam/eam-light-frontend/pull/147

### CODE REVIEW

**Comment by Amfgcp** [comment]
Depends on https://github.com/cern-eam/eam-light-backend/pull/79 and https://github.com/cern-eam/eam-light-frontend/pull/147

File: src/ui/components/checklists/Checklists.js [hunk 1]
```diff
@@ -249,7 +250,8 @@ class Checklists extends Component {
                 <ChecklistEquipment 
                     key={firstChecklist.checkListCode + "_equipment"}
                     equipmentCode={equipmentCode}
-                    equipmentDesc={firstChecklist.equipmentDesc}/>
+                    equipmentDesc={firstChecklist.equipmentDesc}
+                    otherId={eqpMap?.[equipmentCode]}/>
```
Comment (ricardo-j-almeida) [inline src/ui/components/checklists/Checklists.js:254]: If this is supposed to be a map with K,V as equipmentCode,otherId why is it called eqpMap? That might lead people into thinking the equipment object is the value instead

Comment (Amfgcp) [reply src/ui/components/checklists/Checklists.js:254]: Should be solved in: https://github.com/cern-eam/eam-components/pull/121
When I opened this PR I didn't have right to this repo so I was working on a fork in my account.
I will close this PR, sorry for the ping-pong

**Comment by Amfgcp** [comment]
Transitioning to PR: https://github.com/cern-eam/eam-components/pull/121
(When I opened this PR I didn't have right to this repo so I was working on a fork in my account)

**Pull request closed without merging**